packages/ts/react-form/src/index.ts

```diff
@@ -72,6 +75,12 @@ export type UseFormResult<M extends AbstractModel> = Omit<UseFormPartResult<M>,
     update(): void;
   }>;
 
+export type UseFormArrayResult<M extends ArrayModel> = Readonly<{
+  append(item: Value<ArrayItemModel<M>>): void;
+  list: ReadonlyArray<UseFormPartResult<ArrayItemModel<M>>>;
+  remove(index: number): void;
+}>;
+
```

Comment:
I guess, this one remains unused since we've deleted all the code. We can wait until the discussion is finished and then either remove it or leave it here.